### ISSUE TITLE
TST failures on TravisCI with statespace

### ISSUE THREAD

**Issue by josef-pkt** [statsmodels/statsmodels#2837]
#2836 works around the missing gfortran and gets full test results again
only some failures in statespace, 
- 2 small precision problems
- test_sarimax.TestFriedmanStateRegression.test_mle  large failure

https://travis-ci.org/statsmodels/statsmodels/jobs/110699316

```
cython: 0.23.4 (/home/travis/miniconda2/envs/statsmodels-test/lib/python2.7/site-packages/Cython)
numpy: 1.7.1 (/home/travis/miniconda2/envs/statsmodels-test/lib/python2.7/site-packages/numpy)
scipy: 0.12.0b1 (/home/travis/miniconda2/envs/statsmodels-test/lib/python2.7/site-packages/scipy)
pandas: 0.13.0 (/home/travis/miniconda2/envs/statsmodels-test/lib/python2.7/site-packages/pandas)
    dateutil: 2.4.1 (/home/travis/miniconda2/envs/statsmodels-test/lib/python2.7/site-packages/dateutil)
patsy: 0.2.1 (/home/travis/miniconda2/envs/statsmodels-test/lib/python2.7/site-packages/patsy)
```

```
======================================================================
FAIL: statsmodels.tsa.statespace.tests.test_mlemodel.test_fit_misc
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/miniconda2/envs/statsmodels-test/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/travis/miniconda2/envs/statsmodels-test/lib/python2.7/site-packages/statsmodels-0.8.0-py2.7-linux-x86_64.egg/statsmodels/tsa/statespace/tests/test_mlemodel.py", line 198, in test_fit_misc
    assert_almost_equal(res_params, [0,0], 7)
  File "/home/travis/miniconda2/envs/statsmodels-test/lib/python2.7/site-packages/numpy/testing/utils.py", line 452, in assert_almost_equal
    return assert_array_almost_equal(actual, desired, decimal, err_msg)
  File "/home/travis/miniconda2/envs/statsmodels-test/lib/python2.7/site-packages/numpy/testing/utils.py", line 812, in assert_array_almost_equal
    header=('Arrays are not almost equal to %d decimals' % decimal))
  File "/home/travis/miniconda2/envs/statsmodels-test/lib/python2.7/site-packages/numpy/testing/utils.py", line 645, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not almost equal to 7 decimals
(mismatch 50.0%)
 x: array([  6.70331276e-13,   2.76843861e-07])
 y: array([0, 0])
======================================================================
FAIL: statsmodels.tsa.statespace.tests.test_sarimax.TestFriedmanStateRegression.test_mle
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/miniconda2/envs/statsmodels-test/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/travis/miniconda2/envs/statsmodels-test/lib/python2.7/site-packages/statsmodels-0.8.0-py2.7-linux-x86_64.egg/statsmodels/tsa/statespace/tests/test_sarimax.py", line 693, in test_mle
    atol=1e-1, rtol=1e-1
  File "/home/travis/miniconda2/envs/statsmodels-test/lib/python2.7/site-packages/numpy/testing/utils.py", line 1179, in assert_allclose
    verbose=verbose, header=header)
  File "/home/travis/miniconda2/envs/statsmodels-test/lib/python2.7/site-packages/numpy/testing/utils.py", line 645, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Not equal to tolerance rtol=0.1, atol=0.1
(mismatch 100.0%)
 x: array([   0.94494127,    0.3156544 ,  102.73416625])
 y: array([  0.9348486 ,   0.3090592 ,  93.22497257])
======================================================================
FAIL: statsmodels.tsa.statespace.tests.test_sarimax.test_simple_time_varying
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/miniconda2/envs/statsmodels-test/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/travis/miniconda2/envs/statsmodels-test/lib/python2.7/site-packages/statsmodels-0.8.0-py2.7-linux-x86_64.egg/statsmodels/tsa/statespace/tests/test_sarimax.py", line 1787, in test_simple_time_varying
    assert_almost_equal(res.params, [0,0], 7)
  File "/home/travis/miniconda2/envs/statsmodels-test/lib/python2.7/site-packages/numpy/testing/utils.py", line 452, in assert_almost_equal
    return assert_array_almost_equal(actual, desired, decimal, err_msg)
  File "/home/travis/miniconda2/envs/statsmodels-test/lib/python2.7/site-packages/numpy/testing/utils.py", line 812, in assert_array_almost_equal
    header=('Arrays are not almost equal to %d decimals' % decimal))
  File "/home/travis/miniconda2/envs/statsmodels-test/lib/python2.7/site-packages/numpy/testing/utils.py", line 645, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not almost equal to 7 decimals
(mismatch 50.0%)
 x: array([  6.70331276e-13,   2.76843861e-07])
 y: array([0, 0])
```

**Comment by josef-pkt** [comment]
@ChadFulton  can you look at these?

**Comment by ChadFulton** [comment]
Hoping that these will be fixed along with #2839.

**Comment by josef-pkt** [comment]
They also have gone away in current master after fixing mkl in #2846

closing for now
